scripts/set_requester_pays.sh

```diff
@@ -15,23 +15,23 @@ fi
 PROJECT_ID=$1
 USER_EMAIL=$(gcloud config get-value account)
 MEMBER="user:${USER_EMAIL}"
-ROLE="projects/${PROJECT_ID}/roles/RequesterPays"
+ROLE="organizations/386193000800/roles/RequesterPaysToggler"
```

Comment:
Do we need to have an organizations_id, instead of having it there?

Reply:
that's a good question... i think as long as this is for Terra, that's the organization id - but I can make that more explicit here. thank you!